Sources/VLCMediaPlayer.m

```diff
@@ -974,17 +974,11 @@ - (NSInteger)currentAudioPlaybackDelay
 - (void)setEqualizerEnabled:(BOOL)equalizerEnabled
 {
     _equalizerEnabled = equalizerEnabled;
-    if (!_equalizerEnabled) {
-        libvlc_media_player_set_equalizer(_playerInstance, NULL);
-
-        if (_equalizerInstance)
-            libvlc_audio_equalizer_release(_equalizerInstance);
-        return;
-    }
-
-    if (!_equalizerInstance)
+    if (!_equalizerInstance) {
         _equalizerInstance = libvlc_audio_equalizer_new();
```

Comment:
this can return NULL and should be handled

Reply:
You're right, I totally missed it!
I'm not sure how it should be handled? Since it's a setter it feels wrong to have a return value.
Would an NSAssert suffice? 🤔

Reply:
Ideally I would change the return value but that would break api. I think not setting the equalizer to enabled and handle that case in iOS and change the description to mention that would be good.